applications/FluidDynamicsApplication/custom_constitutive/newtonian_3d_law.cpp

```diff
@@ -106,6 +104,11 @@ double Newtonian3DLaw::GetEffectiveViscosity(ConstitutiveLaw::Parameters& rParam
     return rParameters.GetConstitutiveMatrix()(5,5);
 }
 
+double Newtonian3DLaw::ComputeEffectiveViscosity(ConstitutiveLaw::Parameters& rParameters) const {
+    const double effective_viscosity = rParameters.GetMaterialProperties()[DYNAMIC_VISCOSITY];
```

Comment:
You can avoid declaring the variable by doing
```cpp
return rParameters.GetMaterialProperties()[DYNAMIC_VISCOSITY];
```

Reply:
ok, done. Thanks!